beacon_node/beacon_chain/src/beacon_block_streamer.rs

```diff
@@ -19,7 +19,7 @@ use types::{
 };
 
 #[derive(PartialEq)]
-pub enum CheckEarlyAttesterCache {
+pub enum CheckCaches {
```

Comment:
nit: why not use `bool`?

Reply:
We usually try to use enums over bools when we need to pass something into a function. If you end up with two bools as params it can be ambiguous